index.js

```diff
@@ -0,0 +1,67 @@
+import Tasks from './taskFunctions.js';
+import './style.css';
+import refresh from './refresh.svg';
+import enter from './enter-add.svg';
+
+const todoTitle = document.querySelector('.title');
+
+const addToList = document.querySelector('.add-task');
+
+const refreshIcon = document.createElement('img');
+const enterIcon = document.createElement('img');
+refreshIcon.src = refresh;
+enterIcon.src = enter;
+refreshIcon.className = 'refresh';
+enterIcon.className = 'enter';
+todoTitle.appendChild(refreshIcon);
+addToList.appendChild(enterIcon);

```

Comment:
- As per the YAGNI principle, this code could potentially be omitted. You could as well 'wrap it' in a function and call that function instead.